### DISCUSSION TITLE
Alpine Vaultwarden "Admin_Tocken"

### DISCUSSION THREAD

**Discussion by judie07** [tteck/Proxmox#1415] [Q&A] [answered]
Hello, I need help.
All descriptions and helps assume Docker when setting/changing the "Admin_Token", but how to do it without Docker???
==> The admin panel is disabled, please configure the 'ADMIN_TOKEN' variable to enable it

**Comment by tteck** [comment, marked as answer]
![Screenshot 2023-05-12 4 46 05 AM](https://github.com/tteck/Proxmox/assets/21342670/1d4fb6cd-129c-4712-af26-a595fbade173)

**Reply by judie07** [reply]
Those who can read have a clear advantage
sorry for my mistake
